src/main/java/pwr/isa/klama/rentalItem/RentalItemController.java

```diff
@@ -0,0 +1,48 @@
+package pwr.isa.klama.rentalItem;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "api/v1/rentalItem")
+public class RentalItemController {
+
+    private final RentalItemService rentalItemService;
+
+    @Autowired
+    public RentalItemController(RentalItemService rentalItemService) {
+        this.rentalItemService = rentalItemService;
+    }
+
+    @GetMapping
+    public List<RentalItem> getRentalItems() {
+        return rentalItemService.getRentalItem();
+    }
+
+    @GetMapping(path = "{rentalItemId}")
+    public RentalItem getRentalItem(@PathVariable("rentalItemId") Long rentalItemId) {
+        return rentalItemService.getRentalItemById(rentalItemId);
+    }
+
+    @PostMapping
```

Comment:
Dodałbym tutaj specyficzną ścieżkę dodawania - np. '/add` - żeby to było jakkolwiek oddzielone :)